client/ayon_core/tools/publisher/widgets/report_page.py

```diff
@@ -1459,13 +1459,14 @@ class CrashWidget(QtWidgets.QWidget):
     def __init__(self, controller, parent):
         super(CrashWidget, self).__init__(parent)
 
-        main_label = QtWidgets.QLabel("This is not your fault", self)
+        main_label = QtWidgets.QLabel("Error Occurred!", self)
         main_label.setAlignment(QtCore.Qt.AlignCenter)
         main_label.setObjectName("PublishCrashMainLabel")
 
         report_label = QtWidgets.QLabel(
             (
-                "Please report the error to your pipeline support"
+                "Expand the orange error message below to find more info.\n"
+                "You can report the error to your pipeline support"
```

Comment:
If the information is in the orange bar - we should auto-expand it. Not ask the user to do so.

Reply:
you know what.. I'll just remove that part ` "Expand the orange error message below to find more info.\n"`

Reply:
I'll just remove  `"Expand the orange error message below to find more info.\n"` . I think it's redundant. 

I think. It'd be better to have the error message un-expanded because this PR adds message and title to the `KnownPublishError` . and having the error expanded will show the error message.